source/fundamentals/bson.txt

```diff
@@ -210,6 +210,29 @@ Driver will marshal structs using the following rules:
       - Includes an empty ``lastname`` field
       - Stores the ``Address`` field as a nested value
 
+BSON Options
+------------
+
+You can set ``BSONOptions`` to specify how
+the driver marshals and unmarshals BSON on any level, from
+``collection`` to ``Client``.
+
+The following example demonstrates how the ``BSONOptions`` is set using
+the ``Movies`` collection as an example.
+
+.. code-block:: go
+   :copyable: false
+
+   bsonOpts := &options.BSONOptions{
+		UseJSONStructTags: true,
+	}
+	coll := client.Database("sample_mflix").Collection("movies",
+	bsonOpts)
+   
+To learn more about the options and usage of ``BSONOptions``, visit the `API Documentation
+<https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo#example-Connect-BSONOptions>`_
```

Comment:
S: We almost always use source constants (in snooty.toml) to represent API links, making it easier and less error-prone. I would try using a source constant here.